### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by meguiraun**
small fixes

### CODE REVIEW

File: mxcube3/routes/qutils.py [hunk 1]
```diff
@@ -1483,6 +1483,11 @@ def execute_entry_with_id(sid, tindex=None):
         node_id = current_queue[sid]["tasks"][int(tindex)]["queueID"]
 
         node, entry = get_entry(node_id)
+	# in order to fill lims data, we execute first the parent (group_id missing)
+	parent_id = node.get_parent()._node_id
```
Comment (marcus-oscarsson) [inline mxcube3/routes/qutils.py]: Watch out,  indent seems to be off by one ?

Comment (meguiraun) [reply mxcube3/routes/qutils.py]: good catch! thanks

**Comment by meguiraun** [comment]
did I ever mentioned that I hate git? wip now, do not merge

**Comment by marcus-oscarsson** [comment]
:) okey

**Comment by meguiraun** [comment]
thanks, hope you had a nice trip back!

**Pull request merged**